portafly/src/i18n/locales/en/audience/accounts/listing.yml

```diff
@@ -0,0 +1,108 @@
+---
+#--------------------------Path: Audience > Accounts > Listing ------------------------------------#
+
+# Page details
+
+title_page: Accounts | Listing
+title_page_description: Body title for a page aimed to list all the accounts, as well as perform individual or batch actions upon the accounts.
+
+# Top area
+
+create_account_button: Create account
+create_account_button_description: Button link to create an account
+export_accounts_button: Export all accounts
+export_accounts_button_description: Button link to export all the accounts.
+
+# Main area
+
+## Toolbar
+actions_empty_label: Select accounts first to enable actions
```

Comment:
@porueesq curious about why this string is called 'empty'?
Also this string and `actions_dropdownlist` are maybe global strings?

Reply:
Yes, these two can be in the shared file.
_Empty_, meaning no selection was done.